puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -24,6 +24,15 @@ bundle install --without=development --jobs=5 --retry=5
 # we need to install node modules for integration tests (which only run on postgresql)
 if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then
   npm install npm@'<6.0.0' # first upgrade to newer npm
+
+  # Test asset precompile
+  NODE_ENV=production $APP_ROOT/node_modules/.bin/npm install
+  bundle exec rake assets:precompile RAILS_ENV=production
+  bundle exec rake webpack:compile RAILS_ENV=production
+
+  rm -rf public/assets public/webpack package-lock.json
+
+  # Now install all production and dev dependencies
```

Comment:
Do we actually need the dev dependencies on jenkins after the proposed change?

Reply:
Are there not JS unit tests that they are required to run for?

Reply:
I may be mistaken, but I think we only run js unit tests on travis.

Reply:
We still need them, as integration tests will indirectly check the JS works.